packages/block-editor/src/components/rich-text/index.js

```diff
@@ -214,13 +217,27 @@ export function RichTextWrapper(
 
 			const { getBlockAttributes } = select( blockEditorStore );
 			const blockAttributes = getBlockAttributes( clientId );
-			const fieldsList = blockBindingsSource?.getFieldsList?.( {
-				select,
-				context: blockBindingsContext,
-			} );
+			let clientSideFieldLabel = null;
+			const allSources = getBlockBindingsSources();
+			const clientSideSource = allSources[ relatedBinding.source ];
```

Comment:
Can't we use `blockBindingsSource` from further above (which was defined as `getBlockBindingsSource( relatedBinding.source )`? Or is that different? (How?)